extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/deployAzureSQLDBWizard.ts

```diff
@@ -172,4 +189,33 @@ export class DeployAzureSQLDBWizard extends WizardBase<DeployAzureSQLDBWizard, W
 			level: azdata.window.MessageLevel.Error
 		};
 	}
+
+	public async refreshPages() {
+
+		const pageCount = this.wizardObject.pages.length;
+		for (let i = 1; i < pageCount; i++) {
+			this.wizardObject.removePage(this.wizardObject.pages.length - 1);
+			this.wizardObject.pages.pop();
+		}
+		if (instanceOfAzureSQLDBDeploymentProvider(this.resourceProvider)) {
+			this._wizardInfo = this.resourceProvider.azureSQLDBWizard!;
+		} else {
+			return;
```

Comment:
Returning without creating new pages?

Reply:
This is for other deployments that do not have additional pages.

Reply:
Not sure what that's supposed to mean. What other deployments? This is in deployAzureSQLDBWizard so I would expect this to only be used for Azure SQL DB Deployments

Reply:
So SQL DB has 2 providers (out of 3) which only opens a web page. In that case, there won't be any pages to add. Does that make sense?